src/main/resources/org/jenkinsci/plugins/authorizeproject/form/dropdownDescriptorSelector.jelly

```diff
@@ -0,0 +1,73 @@
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <st:documentation>
+    Renders a single &lt;select> control for choosing a Describable.
+    Depending on the currently selected value, its config.jelly will be
+    rendered below &lt;select, allowing the user to configure Describable.
+
+    <st:attribute name="field" use="required">
+      form field name. Used for databinding.
+    </st:attribute>
+    <st:attribute name="title" use="required">
+      Human readable title of this control.
+    </st:attribute>
+    <st:attribute name="lazy">
+      If specified, the additional the variables to be captured. See the @capture of &lt;renderOnDemand> tag.
+    </st:attribute>
+    <st:attribute name="descriptors">
+      Collection that lists up all the valid candidate descriptors.
+      If unspecified, inferred from the type of the field.
+    </st:attribute>
+    <st:attribute name="default">
+      If specified, this will be chosen as the default value in case the current selection is null. The default can be
+      an specific instance or a descriptor e.g.
+      ${descriptor.defaultSettingsProvider} or ${descriptor.defaultSettingsProvider.descriptor}. In the later case, the
+      from input fields will be empty.
+    </st:attribute>
+  </st:documentation>
+  <f:prepareDatabinding/>
+  <j:set target="${attrs}" property="descriptors"
+         value="${attrs.descriptors ?: descriptor.getPropertyType(instance,attrs.field).getApplicableDescriptors()}"/>
+
+  <f:dropdownList name="${attrs.field}" title="${attrs.title}" help="${descriptor.getHelpFile(attrs.field)}">
+    <d:invokeBody/>
+
+    <j:set var="current" value="${instance[attrs.field]}"/>
+    <j:set var="current" value="${current!=null ? current : (default.descriptor!=null ? default : null)}"/>
+    <j:forEach var="descriptor" items="${attrs.descriptors}" varStatus="loop">
+      <f:dropdownListBlock value="${loop.index}" title="${descriptor.displayName}"
+                           selected="${current.descriptor==descriptor || (current==null and descriptor==attrs.default)}"
+                           staplerClass="${descriptor.clazz.name}"
+                           lazy="${attrs.lazy?attrs.lazy+',':''}descriptor,it">
+        <l:ajax>
+          <j:set var="instance" value="${current.descriptor==descriptor ? current : null}"/>
+          <st:include from="${descriptor}" page="${descriptor.configPage}"/>
+        </l:ajax>
+      </f:dropdownListBlock>
+    </j:forEach>
+  </f:dropdownList>
+</j:jelly>
```

Comment:
ROFL: https://issues.jenkins-ci.org/browse/JENKINS-19565 and https://github.com/jenkinsci/jenkins/pull/1184

I'm going to merge https://github.com/jenkinsci/jenkins/pull/1184 ;-)